modules/packages/NetCDF.chpl

```diff
@@ -17,7 +17,29 @@
  * limitations under the License.
  */
 
+/* NetCDF bindings for Chapel
+
+This module implements the C-API for netCDF version 4.6.1.  It is expected
+that higher-level functionality will be built on top of this API similar to
+the functions in the :mod:`HDF5` module.
+
+Compiling with netCDF
+---------------------
+
+In order to compile a Chapel program that uses this module, the netCDF library
+must be installed on the system.  The paths to the ``netcdf.h`` header file
+and netCDF library must be known to the compiler, either by finding them in
+a default search path, or by using the ``-I`` and ``-L`` compiler arguments.
```

Comment:
Bonus points for a code block showing a compilation with the correct `-I`, `-L`, and `-l` flags, similar to other modules with third-party dependencies.